### ISSUE TITLE
Is it possible to run LocalContext on Windows?

### ISSUE THREAD

**Issue by thangckt** [deepmodeling/dpdispatcher#511]
Dear Developers,

I wonder if it is possible to run LocalContext on Windows? Or how to run it

Thanks.

**Comment by njzjz** [comment]
No, `shell` is only designed for `bash` on Linux unless you are interested in making it also work for PowerShell.

I think you can use WSL, though.

**Comment by thangckt** [comment]
hi @njzjz 

Thank you so much for your information.
May we implement a new class for Batch?

**Comment by njzjz** [comment]
Closing because native Windows execution is outside the supported Shell and LocalContext design; WSL remains the recommended path.

Coding agent: Codex
Codex version: codex-cli 0.149.0
Model: gpt-5.6-sol
Reasoning effort: xhigh
